public/schema.json

```diff
@@ -37,9 +37,6 @@
     "/accounts/{account_id}/trial_containers/{trial_container_id}/assets/{asset_id}": {
       "$ref": "refs/paths/accounts_{account_id}_trial_containers_{trial_container_id}_assets_{asset_id}.json"
     },
```

Comment:
We still need the ref below because it included the `get` request for retrieving assets